docs/help-center/first-steps/first-login.mdx

```diff
@@ -94,3 +94,18 @@ With the passphrase, you have access to all your encrypted data and previous cha
 The passphrase cannot be recovered if lost!
 
 :::
+
+### Configure the App Lock
+
+<Tabs groupId="operating-systems">
+  <TabItem value="desktop" label="Desktop">
+
+  Currently there is no app lock on Desktop.
+
+  </TabItem>
+  <TabItem value="mobile" label="Mobile" default>
+
+  You may get prompted to setup the app lock if your device has noch lock screen enabled. You can avoid configuring an app lock by enabling the device lock in your system settings. Otherwise please follow the steps to setup your app lock. These are mostly the same as when configuring them in the [**Settings**](../your-account/account-settings#screen-lock), but you are prompted for them automatically if an app lock is required. You can reset the app lock by logging out in case you ever forget your pin.
```

Comment:
```suggestion
  You may get prompted to setup the app lock if your device has no lock screen enabled. You can avoid configuring an app lock by enabling the device lock in your system settings. Otherwise please follow the steps to setup your app lock. These are mostly the same as when configuring them in the [**Settings**](../your-account/account-settings#screen-lock), but you are prompted for them automatically if an app lock is required. You can reset the app lock by logging out in case you ever forget your pin.
```